src/Components/Patient/PatientRegister.tsx

```diff
@@ -1683,7 +1683,21 @@ export const PatientRegister = (props: PatientRegisterProps) => {
                                 {...field("occupation")}
                                 label="Occupation"
                                 placeholder="Select Occupation"
-                                options={occupationTypes}
+                                options={occupationTypes
+                                  .sort((a, b) => a.text.localeCompare(b.text))
+                                  .filter((obj) => obj.text !== "Others")
+                                  .filter(
+                                    (obj) => obj.text !== "Not Applicable",
+                                  )
+                                  .filter((obj) => obj.text !== "Don't Know")
+                                  .concat(
+                                    { id: 6, text: "Others", value: "OTHERS" },
+                                    {
+                                      id: 32,
+                                      text: "Not Applicable",
+                                      value: "NOT_APPLICABLE",
+                                    },
+                                  )}
```

Comment:
This seems to be adding unnecessary complications for a simple choice field. 

It'd be cleaner and simpler to just manually sort the constants rather than performing `.sort`, `.filter` multiple times and doing `.concat`.

<img width="945" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/2b2f4501-dbb7-414a-9da3-cf4f36d33495">

Reply:
Hint: You can just run the logic on a node repl, print the output and copy paste it in the constants file 😄

Reply:
Yeah initially had a similar thought😅 but was confused with id key. After manual sort should id's be rearranged or kept same?

Reply:
`id` should be kept same as it is used while making API calls to the backend.

Reply:
> `id` should be kept same as it is used while making API calls to the backend.

Fixed. Please review